### PR TITLE
Add DisplayName(), make the Prestige field a single value, and make project files portable

### PR DESCRIPTION
**Pull request by SteveBenz**
There are three commits in this PR.  They're all grouped up because that's how I've tested them and (hopefully) you'll want all three or you'll be happy with breaking them out if needed.

The three changes:

1. Removed the hard-coded paths to KSP from the project files to make it easy for other developers to contribute.
2. "Prestige" is presented in the wiki as a single field, but in the code, it's a list.  For me, that generated errors when I tried to specify Prestige.  I'm not sure why it ever was a list; seems kinda wrong.
3. Added "DisplayName()" methods to CelestialBody and Kerbal classes to allow the use of Lingoona extensions.  I also added a "FirstName()" method to Kerbal so that they can be referred to in text as "Jebediah" rather than "Jebediah Kerman".  This string also has Lingoona extensions to it too, so it should only really be used inside of Format calls.

### CODE REVIEW

File: source/CC_RemoteTech/CC_RemoteTech.csproj [hunk 1]
```diff
@@ -36,11 +37,11 @@
   </PropertyGroup>
   <ItemGroup>
     <Reference Include="Assembly-CSharp">
-      <HintPath>..\..\..\..\..\..\..\Program Files (x86)\Kerbal Space Program\KSP_x64_Data\Managed\Assembly-CSharp.dll</HintPath>
+      <HintPath>$(KspInstallationDir)\KSP_x64_Data\Managed\Assembly-CSharp.dll</HintPath>
```
Comment (SteveBenz) [inline source/CC_RemoteTech/CC_RemoteTech.csproj:40]: LinuxGuruGamer suggested a change to PKS which I think you might want to do, which is to add the references in bulk, like so:

```
    <Reference Include="$(KspInstallationDir)\KSP_x64_Data\Managed\Assembly*.dll">
      <Private>False</Private>
    </Reference>
    <Reference Include="$(KspInstallationDir)\KSP_x64_Data\Managed\UnityEngine*.dll">
      <Private>False</Private>
    </Reference>

```

File: source/ksplocations_example.props [hunk 2]
```diff
@@ -0,0 +1,8 @@
+<Project>
+  <PropertyGroup>
+    <!-- Edit these to point to the files as you find them on your machine -->
+    <TextTransformExe>C:\Program Files (x86)\Microsoft Visual Studio\2017\Enterprise\Common7\IDE\texttransform.exe</TextTransformExe>
+    <Pdb2MdbExe>c:\pdb2mdb\pdb2mdb.exe</Pdb2MdbExe>
```
Comment (SteveBenz) [inline source/ksplocations_example.props:5]: This is out of date.  Should be:

`C:\Program Files\Unity\Editor\Data\MonoBleedingEdge\lib\mono\4.5\pdb2mdb.exe`

**Comment by jrossignol** [comment]
I think there was a good reason for prestige to be a list, but I need to
check.  Otherwise everything seems good in concept, will review the changes
later.

**Comment by jrossignol** [comment]
Prestige needs to be a list, changing it will break Field Research.  It's because of the backwards way that KSP generates contracts.  You get a contract generated, and then throw it away if it isn't an acceptable prestige level.  There is no way to say "give me  contract of prestige x".  The easiest way to deal with this ended up just letting prestige be a list.  If you only want one prestige, then you can do `[ Trivial ]`.  If you want to be able to have your contract have more than one prestige, don't decide the prestige up-front.  Instead, put all valid prestiges in a list and then have other factors dependent on the prestige that was "chosen" by KSP (using the Prestige method).  Here's an example from a [Field Research contract](https://github.com/jrossignol/ContractPack-FieldResearch/blob/master/GameData/ContractPacks/FieldResearch/GeologicalStudy.cfg#L30-L37):
```
    DATA
    {
        type = CelestialBody
        uniquenessCheck = CONTRACT_ACTIVE
        hidden = true
        targetBody1 = Prestige() == Trivial ? HomeWorld() : Prestige() == Significant ? @FieldResearch:l2Bodies.Where(cb => cb.HasSurface()).Random() : @FieldResearch:l3Bodies.Where(cb => cb.HasSurface()).SelectUnique()
    }
```

So I'll reject the prestige change, but merge the rest.

**Comment by jrossignol** [comment]
Okay, another fairly big change - I removed the DisplayName() methods as it's inconsistent with how it's handled everywhere else (which is to let it be automatically casted to a string).  Instead I've made sure that the casting consistently keeps/adds the grammar suffixes.  I'll need to test a bit to make sure there isn't anywhere left that would display that directly to the user, but the Localization changes I made should've taken care of that.

**Pull request merged**